lib/main.dart

```diff
@@ -33,14 +34,16 @@ class EndavaProfileApp extends StatelessWidget {
         Paths.LOGIN: (c) => LoginScreen(),
         Paths.CONTRIBUTORS: (c) =>
             ContributorsScreen(repository: 'stpopa/endava_profile_app'),
+        Paths.SUMMARY_SCREEN: (c) => SummaryScreen(),
       },
       home: BlocBuilder<AuthenticationBloc, AuthenticationState>(
         builder: (context, state) {
           if (state is AuthenticatedState) {
             return HomeScreen();
           }
 //          if (state is UnauthenticatedState) {
```

Comment:
i assume this is leftover of debugging )

Reply:
Removing, my bad).